nimbus-core/src/main/java/com/antheminc/oss/nimbus/app/extension/config/DefaultFrameworkExtensionsConfig.java

```diff
@@ -176,4 +184,24 @@ public StaticCodeValueBasedCodeToLabelConverter staticCodeValueBasedCodeToLabelC
 		return new StaticCodeValueBasedCodeToLabelConverter(beanResolver);
 	}
 	
+	@Bean
+	public UnivocityExcelToCSVConverter excelToCsvConverter() {
```

Comment:
Please move all bean confis to a new Config file for converters.

Reply:
Fixed.